include/proxy-wasm/context.h

```diff
@@ -308,6 +312,7 @@ class ContextBase : public RootInterface,
                                string_view /* details */) override {
     return unimplemented();
   }
+  void failStream(WasmStreamType stream_type) override { closeStream(stream_type); }
```

Comment:
Why do we need an alias for `closeStream`?

Reply:
This is to allow an override where we (for example) send back a 500 error instead of just closing the stream.  I am not sure if it is necessary, but give that we were discussing that I decided to make these two separate so that the logical fail could do something other than close immediately.